Change Wiki documenation usage (#32)
esdc-devcop/strategy: guides/tfvc-to-git.md

Taking it a step further to change your documentation to markdown is even better.
That will allow you to track changes through your documentation, and allow you to begin to automate the documentation.

> Wiki's should be used to document client or user information while documenation related to development or deployments should be kept with the source code itself.

Reviewer: ```suggestion
Wikis should be used to document client or user information while documentation related to development or deployments should be kept with the source code itself.
```

"Wikis" seems to be the plural used by Wikipedia. [link](https://en.wikipedia.org/wiki/List_of_wikis)

You should also feel free to not use the _Wiki_ itself at all, if all the documentation is in the source control.

> In a case where you are not using a Wiki, the client and or user documenation should then be published to a website; 

Reviewer: ```suggestion
If you are not using a Wiki, any client or user documentation should be published to a website; 
```

> GitLab & GitHub Pages would be a great method to publish this documentation.

Reviewer: ```suggestion
[GitLab Pages](https://about.gitlab.com/product/pages/) and [GitHub Pages](https://pages.github.com/) are both great services to host this type of documentation.
```
